NightscoutUploadKit/NightscoutPumpEvents.swift

```diff
@@ -93,6 +93,7 @@ public class NightscoutPumpEvents: NSObject {
             enteredBy: eventSource,
             temp: basalRate.rateType == .Absolute ? .Absolute : .Percentage,
             rate: basalRate.rate,
+            percent: basalRate.rate - 100,
```

Comment:
The RateType enum should be changed to be an enum with associated values.  If the temp basal is a percentage value, then only percent should be represented, and an absolute rate not set, and vice versa.

Reply:
Agreed.  Sorry.  I only use percentage, so wasn't paying attention to what that would do on absolute.